docs/design.rst

```diff
@@ -71,23 +71,21 @@ information about persistent identifiers, hence the typical claim
 types are:
 
 - ``is_same_as``
-- ``is_different_than``
+- ``is_variant_of``
 
```

Comment:
Can you please add small description of each predicate, taken from https://github.com/inveniosoftware/claimstore/issues/8#issuecomment-136749882 ?